core/tests/performance_tests/test_utils.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Something here"""
```

Comment:
Needs a real docstring. I'm also concerned with introducing a new test_utils file. This is a bit tricky given that we can't reuse the existing ``test_utils`` file. If we need to have a test base, perhaps ``base.py`` is a more fitting name for this file.